src/Domain/Downtime.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types = 1);
+
+namespace SandwaveIo\RealtimeRegister\Domain;
+
+use Carbon\Carbon;
+
+final class Downtime implements DomainObjectInterface
+{
+    /** @var int */
+    public $id;
+
+    /** @var Carbon */
+    public $startDate;
+
+    /** @var Carbon */
+    public $endDate;
+
+    /** @var string|null */
+    public $reason;
+
+    /** @var Provider */
+    public $provider;
+
+    private function __construct(
+        Carbon $startDate,
+        Carbon $endDate,
+        ?string $reason,
+        Provider $provider
+    ) {
+        $this->startDate = $startDate;
```

Comment:
$this->id is never set.

Reply:
Added: https://github.com/sandwave-io/realtimeregister-php/pull/22/commits/3a455c6b171dc6179598f060e30918f0e4bd0916